### PR TITLE
Add slack integration to sms endpoint

### PR DESCRIPTION
**Pull request by Gathondu**
#### What does this PR do?
Add slack integration that posts errors to a slack channel if the message is not understood by the api.
Rename files.
#### Description of Task to be completed?
When SMS query received that isn't understood, we should post to Slack
#### How should this be manually tested?
run `nosetests -s`

### CODE REVIEW

**Comment by Gathondu** [comment]
@DavidLemayian

**Comment by Gathondu** [comment]
image
@DavidLemayian this is the slack output

File: healthtools_ke_api/views/sms_handler.py [hunk 1]
```diff
@@ -233,6 +237,58 @@ def parse_elastic_search_results(response):
...
+    if SLACK["url"]:
+        response = requests.post(
+            SLACK["url"],
+            data=json.dumps(
+                {
+                    "attachments":
+                        [
+                            {
+                                "author_name": "HealthTools API - Kenya",
+                                "color": "danger",
```
Comment (DavidLemayian) [inline healthtools_ke_api/views/sms_handler.py]: Let's make this warning colour.

File: healthtools_ke_api/views/sms_handler.py [hunk 2]
```diff
@@ -233,6 +237,58 @@ def parse_elastic_search_results(response):
...
+                                "pretext": "[SCRAPER] New Alert for HealthTools API - Kenya SMS Endpoint",
```
Comment (DavidLemayian) [inline healthtools_ke_api/views/sms_handler.py]: Let's make this:
"[SMS] Could not find a result for this SMS."

File: healthtools_ke_api/views/sms_handler.py [hunk 3]
```diff
@@ -233,6 +237,58 @@ def parse_elastic_search_results(response):
...
+                                        },
+                                    {
+                                        "title": "Time",
+                                        "value": "{}".format(datetime.now().strftime("%Y-%m-%d %H:%M:%S")),
+                                        "short": True},
+                                    {
+                                        "title": "Severity",
+                                        "value": message,
+                                        "short": True
+                                        }
```
Comment (DavidLemayian) [inline healthtools_ke_api/views/sms_handler.py]: Remove Machine Location, time and severity.

**Review by DavidLemayian — CHANGES_REQUESTED** [review]

File: healthtools_ke_api/views/sms_handler.py (hunk 1 above)
Comment (Gathondu) [reply healthtools_ke_api/views/sms_handler.py]: @DavidLemayian that's Slack's API. Those are the keys for formatting the message to slack.
Refer to 👉 [Webhook message formatting](https://api.slack.com/docs/message-attachments)

File: healthtools_ke_api/views/sms_handler.py (hunk 2 above)
Comment (Gathondu) [reply healthtools_ke_api/views/sms_handler.py]: 👍

File: healthtools_ke_api/views/sms_handler.py (hunk 3 above)
Comment (Gathondu) [reply healthtools_ke_api/views/sms_handler.py]: 👍

Comment (Gathondu) [inline healthtools_ke_api/views/sms_handler.py]: image
@DavidLemayian better?

**Comment by Gathondu** [comment]
@DavidLemayian better?
image

File: healthtools_ke_api/views/sms_handler.py (hunk 1 above)
Comment (DavidLemayian) [reply healthtools_ke_api/views/sms_handler.py]: @Gathondu Perhaps we're speaking past each other.

I'd want this `"color": "danger",` to be `"color": "warning",`.

Is this against Slack's API?

Comment (Gathondu) [reply healthtools_ke_api/views/sms_handler.py]: 🤦‍♂️  🤣  That is what you meant. Sure. @DavidLemayian

Comment (Gathondu) [reply healthtools_ke_api/views/sms_handler.py]: @DavidLemayian Changed.
image

File: healthtools_ke_api/views/sms_handler.py [hunk 4]
```diff
@@ -233,6 +232,44 @@ def parse_elastic_search_results(response):
...
+                                    {
+                                        "title": "Message",
+                                        "value": message,
+                                        "short": False
+                                        }
+                                    ]
+                                }
+                            ]
+                    }),
+            headers={"Content-Type": "application/json"})
```
Comment (DavidLemayian) [inline healthtools_ke_api/views/sms_handler.py]: I was about to let it go until I couldn't... Let's fix the indentation here to follow [style guide](https://google.github.io/styleguide/pyguide.html#Indentation).

**Review by DavidLemayian — CHANGES_REQUESTED** [review]

File: healthtools_ke_api/views/sms_handler.py (hunk 4 above)
Comment (Gathondu) [reply healthtools_ke_api/views/sms_handler.py]: @DavidLemayian 😮 I totally missed that....fixed it. 👍

**Pull request merged**